### DISCUSSION TITLE
bugfixes for customtube 1.2.0.1 suggestions

### DISCUSSION THREAD

**Discussion by bangbangyouall** [lightbeam24/CustomTube#67] [General]
i use edge, so this may not affect anybody else.

-the 2014 layout overlaps the player right a little bit, so you can't see a bit of the video.
-the 2013 layouts have cut off suggestions on the right, making only half the thumbnail seeable.

i haven't seen these be documented in 1.2.0 changelog, so sorry if this is a bit outdated. 1.1.0.3 is what we stablers are currently using.

**Comment by lightbeam24** [comment]
1. CustomTube Stable 1.2.0 is out on Firefox (and is Pending approval on the Chrome Web Store)

2. CustomTube Beta 1.2.0.1 is an outdated Beta. The latest is 1.2.0.3, which released today.

Also could you send screenshots of these issues? Not sure what you're talking about.

**Reply by bangbangyouall** [reply]
![issue 1](https://github.com/lightbeam24/CustomTube/assets/109046841/1b1e28b7-4ff6-43ec-89fd-62933ad4f728)
(i did this on a livestream so you could see what i mean!)
![issue 2](https://github.com/lightbeam24/CustomTube/assets/109046841/9898be74-3934-45f0-a322-802d3382590c)
(this happens on all 2013s btw)

also the like button count is modern & retun dislike support is bugged (this is screenshots from 1.2.0 release btw)

**Comment by lightbeam24** [comment]
Okay so it looks like you have the new "rolling like count" experiment, which breaks some things. I still have to fix that yet.

What settings have you changed in that second screenshot? That doesn't happen to me on Chrome or Firefox (Edge should be no different, though I guess I could test later).

**Reply by bangbangyouall** [reply]
i haven't changed anything in the settings, i even set it back to default values but it still hides part of the right

also rainbow subscribe button is still showing

(btw can you also add sponsorblock support for player)

**Reply by lightbeam24** [reply]
What other extensions/scripts do you have installed? Try disabling them.

> (btw can you also add sponsorblock support for player)

If I get to it.

**Reply by bangbangyouall** [reply]
it was unhook

2014 still does the thing though

**Reply by bangbangyouall** [reply]
you know what? imma start over
